test/new-e2e/pkg/environments/ecs.go

```diff
@@ -18,9 +17,8 @@ import (
 
 // ECS is an environment that contains a ECS deployed in a cluster, FakeIntake and Agent configured to talk to each other.
 type ECS struct {
-	AwsEnvironment *aws.Environment
-	ClusterName    pulumi.StringInput
-	ClusterArn     pulumi.StringInput
+	ClusterName pulumi.StringInput
+	ClusterArn  pulumi.StringInput
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
We should be able to remove also these, we don't export them, I don't think we need them in the Env